src/storage.js

```diff
@@ -0,0 +1,28 @@
+function update(tasks) {
+  window.localStorage.setItem('tasks', JSON.stringify(tasks));
+}
+
+function load() {
+  let list = [{
+    title: 'first',
+    completed: false,
+    index: 1,
+  },
+  {
+    title: 'second',
+    completed: false,
+    index: 2,
+  },
+  {
+    title: 'third',
+    completed: false,
+    index: 3,
+  }];
+
+  if (JSON.parse(localStorage.getItem('tasks'))) {
+    list = JSON.parse(localStorage.getItem('tasks'));
+  }
+  return list;
+}
+
+export { update, load };
```

Comment:
- [ ] As pointed out by the previous reviewer, the logic for marking a task as completed should be in a different file than the index.js, and in this case, the only js file available apart from the index.js is the storage.js which means this is probably where we should have the logic for marking the tasks as completed as mentioned in the project requirements below 👇 
![image](https://user-images.githubusercontent.com/55465133/134559421-c891e2f9-72a5-4de5-b80c-783cbee6db1c.png)

Kindly let's keep the logic for status update (completed: true/false) in a different file like even in this one (storage.js) 😉